src/commonMain/kotlin/com/liftric/cognito/idp/IdentityProviderClient.kt

```diff
@@ -25,24 +27,18 @@ open class IdentityProviderClient(region: String, clientId: String) : IdentityPr
     }
     private val configuration = Configuration(region, clientId)
     private val client = HttpClient {
+        expectSuccess = true
         /**
          * When referencing members that are in the
          * IdentityProvider's scope, assign them to
          * a new variable in this scope. Needed to
          * avoid [InvalidMutabilityException] in iOS.
          */
         val configuration = configuration
-        val json = json
-        install(JsonFeature) {
-            serializer = KotlinxSerializer(json)
-            acceptContentTypes = listOf(
-                ContentType.parse(Header.AmzJson),
```

Comment:
I was at first a bit irritated but I guess it is unneeded anymore due to `contentType(ContentType.parse(Header.AmzJson))`.